### PR TITLE
Changed FTP site used for obtaining Fermi spacecraft pointing files.

### PR DESCRIPTION
**Pull request by aringlis**
The old version of `fermi.download_weekly_pointing_file()` was using provisional planning files for the spacecraft pointing information, rather than the final versions. This PR changes the routine to download the files from the correct FTP site.

### CODE REVIEW

File: sunpy/instr/fermi.py [hunk 1]
```diff
@@ -40,59 +40,39 @@ def download_weekly_pointing_file(date):
...
+    
     # find out which file to get based on date
-    # earliest file in the FERMI server is for mission week 23,
-    # beginning 2008 November 6.
-    weekly_file_start=parse_time('2008-11-06')
-    base_week=23
+    # earliest full file in the FERMI server is for mission week 10,
+    # beginning 2008 August 7.
+    weekly_file_start=parse_time('2008-08-07')
+    base_week=10
```
Comment (wafels) [inline sunpy/instr/fermi.py]: Style only - please ensure that there is one space on either side of the equal signs (no spaces when defining function keywords).

File: sunpy/instr/fermi.py [hunk 2]
```diff
@@ -40,59 +40,39 @@ def download_weekly_pointing_file(date):
...
-    # find out the rest of the file name. Need the year and the day-in-year for
-    # start and end of file
-    start_date = weekly_file_start + datetime.timedelta(weekdiff*7)
-    start_year_str = str(start_date.year) + '-01-01'
-    start_str = start_date.strftime('%Y%j')
-
-    # now end string
-    end_date = weekly_file_start + datetime.timedelta((weekdiff+1)*7)
-    end_str = end_date.strftime('%Y%j')
+    weekstr = ('%03.0f' % week)
```
Comment (dpshelio) [inline sunpy/instr/fermi.py]: can we change this to the `'{:3.f}'.format(week)` syntax?

File: sunpy/instr/fermi.py [hunk 3]
```diff
@@ -40,59 +40,39 @@ def download_weekly_pointing_file(date):
...
-        try:
-            resp = urllib2.urlopen(full_fname)
-            exists = True
-        except:
-            urllib2.HTTPError
-            exists = False
-        # if the file exists then exit and retain this filepath
-        if exists:
-            break
+    full_fname = fbasename + weekstr + '_p202_v001.fits'
```
Comment (dpshelio) [inline sunpy/instr/fermi.py]: please, change also to the string format way

File: sunpy/instr/fermi.py [hunk 4]
```diff
@@ -40,59 +40,39 @@ def download_weekly_pointing_file(date):
...
+        exists = False
 
     # if no matches at all were found, then the pointing file doesn't exist
     if not exists:
         raise ValueError('No Fermi pointing files found for given date!')
 
     # download the file
-    pointing_file_url=full_fname
-    destination=os.path.join(tmp_dir,rest_of_filename)
+    pointing_file_url=base_url + full_fname
```
Comment (dpshelio) [inline sunpy/instr/fermi.py]: if you define this `pointing_file` before line 63, then you can use that variable in 63 instead of defining the url again

**Comment by Cadair** [comment]
@aringlis Are you going to update this or should I close it and you can reopen when you get a chance to work on it?

**Comment by aringlis** [comment]
@Cadair yes I will update it, hopefully by tomorrow. Looks like there are merge conflicts now, doh.

File: sunpy/instr/fermi.py (hunk 3 above)
Comment (aringlis) [inline sunpy/instr/fermi.py]: @dpshelio I'm not sure how you prefer this formatted?

Comment (dpshelio) [inline sunpy/instr/fermi.py]:
```python
full_fname = '{base}_{week:3.f}_p202_v001.fits'.format(base=fbasename, week=week)
```

Comment (aringlis) [inline sunpy/instr/fermi.py]: Yikes.

Ok, I will do this, but that looks way less readable to me!

**Pull request closed without merging**